### ISSUE TITLE
Accessibility Issues -- Keyboard use limited in Read Mode

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#276]
(Also of interest to @nadinelevinson @davidtuckett)

**Accessibility Checklist from a [website](https://www.paciellogroup.com/web-accessibility-checklist-and-wcag-guidelines/?gclid=Cj0KCQiAmfmABhCHARIsACwPRAADzlpH-MEue7pb8b8GAgL3mNupat-8nux51nmvmwQQGmIFV_bO-tUaAmngEALw_wcB&ef_id=Cj0KCQiAmfmABhCHARIsACwPRAADzlpH-MEue7pb8b8GAgL3mNupat-8nux51nmvmwQQGmIFV_bO-tUaAmngEALw_wcB:G:s&utm_source=Adwords):**
- Below is a checklist of some of the most common things to look for in order to conform to WCAG 2.0 and WCAG 2.1:
   - Prioritize simplicity and ease of consumption when creating content in ways that will not sacrifice information or structure
   - Provide text alternatives for visual images that communicate relevant information Provide captions or audio descriptions for videos and audio
   - **Ensure content is fully accessible using only a keyboard**
   - Offer markers or other aids to help users understand where they are during the entire experience and find what they need to
   - Avoid design layouts that are known to cause seizures
   - Programmatically determine the language of the page
   - Include form input error identification and description in text format
   - Use clean markup languages: elements with start and end tags and are properly nested; maintain unique IDs and attributes
   - Enable reflow such that a user need not scroll in two dimensions after enlarging text
   - Utilize a 3:1 color contrast ratio for text

**Issue** Pressing tab moves you from item to item, in a left to right sequence.  But the only way I can figure out to get in a specific "pane" to start tabbing, is by clicking with the mouse.  How can you do it with the keyboard only?

- Here's what I can see **you can do**
   - You can press enter to activate the selected item. 
   - Pressing Spacebar checks and unchecks the item if you're on the checkbox square
   - When a combo box is selected, down and up arrows work fine to move among the items
   - Pressing Enter in the combo box pops it up/pulls it down to see options, and then arrows work there too

- **Potential Accessibility Issues**
   - How can you see where you are starting?  
   - How can you select where you are starting?
   - How can you move more quickly among the tab groups and infocards without tabbing all the way down them.
     - Possibly a specific keyboard shortcut for each group?
     - Using a specific keyboard shortcut that is in the group?  See a list of keyboard shortcuts/targets below, some of which I've discussed with Alan
   - How would you:
       - Check 2 or more refine boxes
          -  When I tab to one and press space to select it, it takes effect immediately and my keyboard cursor is no longer in the refine pane.
       - Select links in the text.  (When tabbing in the document pane, after the author info, goes right to the biblio)
       - Select glossary items in the text
       - display the info in the (i) author information box
       - select the related documents link on items in the read pane (tabbing doesn't go there)

- Theme dependent issues
   -  In most themes, I see an obvious box around Read / Search / View etc. when I manage to tab to them in most themes
            -  This is not visible in the High Contrast Theme
   - The dark theme has some similar issues when you tab around, though not for the items mentioned above.

- Other Individual Issues
   - Tab to the document title...enter does nothing
   - 
When on the read list, when on one of the items, it would be good to have down arrow and up arrow move directly to the item below or above.  E.g., on the biblio entry, move to the next/prev, and on a icon for checkbox/favs/read later, go down or up there too.  That can save a lot of tabbing, and lets the user quickly walk up or down the list of documents and browse them, much nicer than having to tab.

List of possible direct keyboard accelerator shortcuts moved to a separate issue since this one is long.

**Comment by bakerac4** [comment]
We fixed direct issues you mentioned. Otherwise it seems to be working as expected. I believe navigation is following standard protocols, and with the other story we have also added shortcuts to improve upon this for people who want to use just the keyboard
